nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -4,6 +4,7 @@ collision_monitor:
     odom_frame_id: "odom"
     cmd_vel_in_topic: "cmd_vel_raw"
     cmd_vel_out_topic: "cmd_vel"
+    collision_monitor_state_topic: "collision_monitor_state"
```

Comment:
How about to name parameter just `state_topic` (and for all variable names ditto)?